TestApp/update-npm-packages.sh

```diff
@@ -9,18 +9,22 @@ npm pack ../appcenter
 npm pack ../appcenter-analytics
 npm pack ../appcenter-crashes
 npm pack ../appcenter-link-scripts
-# workaround for macs on arm64
-npm install react-native-fs --save --legacy-peer-deps
 
 echo "Installing appcenter* packages..."
 npm install appcenter*.tgz
 
+echo "workaround for macs on arm64"
```

Comment:
do we need to "echo" it? and why did you move it?

Reply:
Echo helps to see at which step the script fails if so. I moved it because on this step we already have our package versions updated (archives with 4.2.0 packages) and `npm install` tries to install previous appcenter versions (4.1.0 in this case, they are set in the package.json at this moment) and can't find them.